shared/src/bundles/users/validation-schemas/user-details-update.validation-schema.ts

```diff
@@ -0,0 +1,74 @@
+import joi from 'joi';
+
+import {
+    CountryList,
+    EmploymentType,
+    EnglishLevel,
+    JobTitle,
+    NotConsidered,
+    PreferredLanguages,
+} from '../enums/enums.js';
+import { type UserDetailsUpdateRequestDto } from '../types/types.js';
+
+const userDetailsUpdate = joi.object<UserDetailsUpdateRequestDto, true>({
+    id: joi.string().trim().required(),
```

Comment:
I'm not sure if id should be required. 

Since atm our token only contains user id, which we will use to find this particular user's user details entity.
Also, I don't think there will be a point in time where we will change a user's user details entity's id.

This is my opinion though. What do you think?

Reply:
I did as you said. 
But now I think that user can have 2 or more profiles to search for jobs. 
For example, find as a js developer and another as a project manager.
But my opinion differs from reffered platforms.